charon/tests/cargo/dependencies.out

```diff
@@ -1,6 +1,29 @@
 # Final LLBC before serialization:
```

Comment:
Can you update the test files in the same commit that caused the change please? This makes it easier to follow what's changing.

Reply:
Ok. I may kinda mess the history like this when I do the rebase.